kie-dmn/kie-dmn-backend/src/main/java/org/kie/dmn/backend/marshalling/v1_6/xstream/ArtifactConverter.java

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
```suggestion
/*
```
Please do it for all introduced classes